.vscode/settings.json

```
@@ -0,0 +1,5 @@
+{
+  "markdown.styles": [
+    "public/vscode_markdown.css"
```

```suggestion
    ".vscode/vscode_markdown.css"
```
I don't think this needs to shipped with the website build. so I would just put it in the .vscode folder

---

It was deliberate to make it public, in case we want to add the VSCode config to all pipelines (which I assume we might want to at some point).

Even if we don't do that, we could have it as a documented thing that people can add to their global VSCode configuration so that it works wherever.

In both cases it's useful to have the file public.